tests/test_tracer.py

```diff
@@ -66,13 +68,15 @@ def _make_cake():
     assert make.span_id
     assert make.parent_id is None
     assert make.trace_id
+    eq_(make.get_tag(system.PID), str(getpid())) # Root span should contain the pid of the current process
```

Comment:
I think that since we're working on this test module, we can start splitting this huge test in small pieces. I'd say to create a new test with just the `getpid()` check, other than adding more and more stuff on this one. Makes sense?